dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnector.java

```diff
@@ -156,9 +157,20 @@ String getFormatName() {
   @Override
   public Handle open(@Nonnull ConnectorArguments arguments) throws Exception {
     Driver driver = newDriver(arguments.getDriverPaths(), "oracle.jdbc.OracleDriver");
-    DataSource dataSource =
-        new SimpleDriverDataSource(driver, buildUrl(arguments), buildProperties(arguments));
-    return new JdbcHandle(dataSource);
+    String url = buildUrl(arguments);
+    DataSource dataSource = new SimpleDriverDataSource(driver, url, buildProperties(arguments));
+    try {
+      return new JdbcHandle(dataSource);
+    } catch (Exception e) {
+      throw new IOException(
+          String.format(
+              "Failed connecting to the Oracle database on %s URL.\n"
```

Comment:
nit: I think `URL` world is not necessary here, as it is obvious from format of value which will be pasted there.

Reply:
👍